### DISCUSSION TITLE
Missing tab head in multiple sheets

### DISCUSSION THREAD

**Discussion by gaglioffo** [spatie/simple-excel#118] [Q&A]
Hi,
when I create multiple sheet I lose the first "tab field" line. It missing because **processingFirstRow** become false after the first loop.

Example of code
```
$this->writer
           ->addNewSheetAndMakeItCurrent()
           ->setName('New Sheet');
...
$this->stream->addRow($table);
```

Probably an easy approach it's to edit the following function
```
public function setHeaderStyle($style) 
{
        $this->headerStyle = $style;

        **$this->processingFirstRow = true;**

        return $this;
}
```

and the new code become 
```
$this->writer
            ->addNewSheetAndMakeItCurrent()
            ->setName('New Sheet');

**$this->stream->setHeaderStyle($stile);** // <- force to set old or new head style
...
$this->stream->addRow($table);
```

Does anyone have a different approach or a smart solution to suggest?

Thanks a lot

**Comment by red-freak** [comment]
I had the same issue, so a created a PR to solve the problem:
https://github.com/spatie/simple-excel/pull/124
